std/algebra/emulated/sw_bn254/precomputations.go

```diff
@@ -50,6 +54,16 @@ func (p *Pairing) computeLines(Q *g2AffP) lineEvaluations {
 		}
 	}
 
+	// check Q is on G2 subgroup:
+	// [r]Q == 0 <==> [6x₀+2]Q + ψ(Q) + ψ³(Q) = ψ²(Q).
+	// At this point Qacc = [6x₀+2]Q
```

Comment:
Maybe just add a comment to reference to [computeG2ShortVector] method to allow reviewers see that we're actually doing exactly the same thing.

Reply:
Actually `computeG2ShortVector` is not the exact same vector. The bottleneck there is `[x₀]Q` but inside `computeLines ` I changed it so that the bottleneck scalar multiplication is the same. The two vectors are equivalent though for BN254 (because of some congruences satisfied by the curve seed). I'll add a comment.